src/library-authoring/list-libraries/LibraryListPage.jsx

```diff
@@ -198,18 +185,53 @@ export class LibraryListPage extends React.Component {
         <div className="wrapper-content wrapper">
           <section className="content">
             <article className="content-primary" role="main">
-              <LibraryIndexTabs />
               {showForm
               && <LibraryCreateForm hideForm={this.hideForm} />}
-              <ul className="library-list">
-                {libraries.data.map((library) => (
-                  <li key={library.id} className="library-item">
-                    <LibraryListItem library={library} />
-                  </li>
-                ))}
-              </ul>
-              {libraries.count > 0
+              {libraries.count !== 0
                 ? (
+                  <ul className="library-list">
+                    {libraries.data.map(library => (
+                      <Card
+                        isClickable
+                        key={library.id}
+                        className="library-item"
+                        onClick={() => this.props.history.push(library.url)}
+                      >
+                        <Card.Header
+                          className="library-title"
+                          title={library.title}
+                        />
+                        <div className="library-metadata">
+                          <span className="library-org metadata-item">
+                            <span className="value">{library.org}</span>
+                          </span>
+                          <span className="library-slug metadata-item">
+                            <span className="value">{library.slug}</span>
+                          </span>
+                        </div>
+                      </Card>
+                    ))}
+                  </ul>
+                ) : (
+                  <EmptyPage
+                    heading={intl.formatMessage(emptyPageMessages['library.list.empty.heading'])}
+                    body={intl.formatMessage(emptyPageMessages['library.list.empty.body'])}
+                  >
+                    <ActionRow className="action-row">
+                      <Button
+                        variant="outline-primary"
+                        size="lg"
+                        className="action-btn"
+                        onClick={this.showForm}
+                      >
+                        <Icon src={Add} />
+                        {intl.formatMessage(emptyPageMessages['library.list.empty.new.library'])}
+                      </Button>
+                    </ActionRow>
+                  </EmptyPage>
+                )}
+              {libraries.count > 1
```

Comment:
Is `1` a correct value here? I thought it might be something like `page_size`

Reply:
Oh, so this is a number of **pages**? I thought it was a number of **libraries** and we shouldn't show pagination if there are less than two pages available

Reply:
I'm confused)) this is a parameter equal `libraries.data.length`
If the second option is more readable I can change it

Reply:
Let's wait for @ihor-romaniuk @UvgenGen or @monteri reactions on this)

Reply:
Does this `libraries.count` mean page count or libraries count?

Reply:
libraries count

Reply:
you are right, replaced to `paginationOptions.pageCount > 1`
I didn't read correctly the description for issue.